src/pages/Swap/AdvancedTradingViewBox/AdvancedSwapMode/AdvancedSwapMode.component.tsx

```diff
@@ -68,6 +69,8 @@ export const AdvancedSwapMode = ({ children }: PropsWithChildren) => {
     ? tradesWrapper?.current?.clientHeight - TradesWrapperHeight.PADDING
     : TradesWrapperHeight.MIN
 
+  const getTransactions = () => transactions.filter(tx => (onlyOpen ? tx.status === 'open' : true))
```

Comment:
What's the rationale for this?

Reply:
I am not sure I understand the question, that is a function that returns filtered transaction, open in case we select open or all otherwise.